GhcEvents.hs

```diff
@@ -13,8 +13,8 @@ import GHC.RTS.Events.Analysis.Capability
 import System.Environment (getArgs)
 import Data.Either (rights)
 import qualified Data.Map as M
-import System.IO (IOMode(ReadMode), openBinaryFile)
-import System.Exit (die)
+import System.IO
+import System.Exit
```

Comment:
Why did you change these to non-selective imports? I see the case for `System.Exit` but not sure about `System.IO`.